lib/src/publisher.rs

```diff
@@ -24,11 +24,17 @@ use tokio::{sync::Mutex, time as timer};
 use tokio_util::udp::UdpFramed;
 
 #[derive(Debug)]
+
+/// Structure to tack a [Subscription](super::subscriber::Subscription)
```

Comment:
should `tack` be `track`? Also, I wouldn't mention that it's a Structure, that will be self evident in the docs already.